src/main/java/cz/startnet/utils/pgdiff/PgDiffTables.java

```diff
@@ -309,7 +309,7 @@ private static void addModifyTableColumns(final List<String> statements,
 
             if (!oldColumn.getType().equals(newColumn.getType())) {
                 statements.add("\tALTER COLUMN " + newColumnName + " TYPE "
-                        + newColumn.getType() + " /* "
+                		+ newColumn.getType() + " USING " + newColumnName  + "::" + newColumn.getType() + " /* "
```

Comment:
This doesn't look like it belongs in this PR -- what does `USING` have to do with `IF (NOT) EXISTS` ?

Reply:
This is for use  #148 use using column::type to cast the type of column
I included this a long time ago in my branch

Reply:
okay :)